File: ServerPart/ServerPart.Data/Models/AuthModels/Account.cs
using ServerPart.Data.Enums;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace ServerPart.Data.Models.AuthModels
{
    public class Account
    {
        public int Id { get; set; }

        public string Login { get; set; }

        public string Password { get; set; }

        public string CarNumber { get; set; }

        public string Email { get; set; }

        public int ConfirmEmail { get; set; }

        public RoleEnum RoleId { get; set; }

        public int WalletId { get; set; }

        public Wallet Wallet { get; set; }

        public DateTime CreateDate { get; set; }

        public DateTime LastLogin { get; set; }
    }
}

File: ClientPart/src/app/Models/Account.ts
import {Wallet} from './Wallet';

export class Account {
  id: number;
  login: string;
  carNumber: string;
  email: string;
  password: string;
  role: any;
  roleId: any;
  confirmEmail: number;
  walletId: number;
  wallet: Wallet = new Wallet();
  createDate: Date;
  lastLogin: Date;
}

export enum RoleEnum {
  User = 1,
  Admin = 2,
  Dispatcher = 3,
}
